core/src/main/scala/com/criteo/cuttle/Cuttle.scala

```diff
@@ -20,6 +20,9 @@ class CuttleProject[S <: Scheduling] private[cuttle] (
   ) = {
     val xa = Database.connect(databaseConfig)
     val executor = new Executor[S](platforms, xa)(retryStrategy)
+
+    WaitingTimesMonitor.start(executor, xa)
```

Comment:
Its should be automatically started on Executor create. The whole WaitingTimes monitoring stuff should not leak outside of Executor.